### PR TITLE
5 get /api/reviews/review

### PR DESCRIPTION
**Pull request by CharlieH94**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -1,84 +1,123 @@
...
+            expect(review).toHaveProperty("title", expect.any(String));
+            expect(review).toHaveProperty("review_id", expect.any(Number));
+            expect(review).toHaveProperty("category", expect.any(String));
+            expect(review).toHaveProperty("review_img_url", expect.any(String));
+            expect(review).toHaveProperty("created_at", expect.any(String));
+            expect(review).toHaveProperty("votes", expect.any(Number));
+            expect(review).toHaveProperty("designer", expect.any(String));
+            expect(review).toHaveProperty("comment_count", expect.any(Number));
+          });
+          expect(reviews[4].comment_count).toBe(3);
```
Comment (Slyose) [inline __tests__/app.test.js:52]: How come you're checking this specific review for 3 comments?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -1,84 +1,123 @@
...
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("owner", expect.any(String));
+            expect(review).toHaveProperty("title", expect.any(String));
+            expect(review).toHaveProperty("review_id", expect.any(Number));
+            expect(review).toHaveProperty("category", expect.any(String));
+            expect(review).toHaveProperty("review_img_url", expect.any(String));
+            expect(review).toHaveProperty("created_at", expect.any(String));
+            expect(review).toHaveProperty("votes", expect.any(Number));
+            expect(review).toHaveProperty("designer", expect.any(String));
+            expect(review).toHaveProperty("comment_count", expect.any(Number));
```
Comment (Slyose) [inline __tests__/app.test.js:50]: Could be made more concise with a method like toMatchObject, but valid as is

**Review by Slyose — APPROVED** [review]
Looks good! Tests can be made more concise, but great overall

File: __tests__/app.test.js (hunk 1 above)
Comment (CharlieH94) [reply __tests__/app.test.js:52]: Hannah suggested optionally adding a test on one particular object to check the comment count is correct and as expected, in order to check the SQL worked correctly

File: __tests__/app.test.js (hunk 2 above)
Comment (CharlieH94) [reply __tests__/app.test.js:50]: Cool, I'll leave as is on this one and use toMatchObject in later tests so have examples of both in that case

**Pull request merged**